vmm/src/vm.rs

```diff
@@ -536,6 +538,8 @@ pub struct Vm {
     #[cfg(all(feature = "kvm", target_arch = "x86_64"))]
     hypervisor: Arc<dyn hypervisor::Hypervisor>,
     stop_on_boot: bool,
+    #[cfg(target_arch = "aarch64")]
+    flash_memory: Option<FlashMemory>,
```

Comment:
I think this should be part of the `MemoryManager`.

Reply:
Or `DeviceManager` even!

Reply:
Yes that works too!

Reply:
OK, now that it's a flash device, I will move it `DeviceManager`.

How about `FlashMemory` itself? Is it better to put it in `devices` crate, or keep it aarch64 specific in `arch`?

Reply:
@michael2012z There are architecture specific devices in the `devices/legacy` crate

Reply:
Now the device is under `devices/legacy/` folder.